pkg/spi-shared/httptransport/auth.go

```diff
@@ -23,7 +23,7 @@ import (
 // http client
 type authenticatingRoundTripperContextKeyType struct{}
 
-var authenticatingRoundTripperContextKey = authenticatingRoundTripperContextKeyType{}
+var AuthenticatingRoundTripperContextKey = authenticatingRoundTripperContextKeyType{}
```

Comment:
I need to public this just for the test. I don't like it. Any better idea?

cc: @metlos